index.html

```diff
@@ -25,23 +25,20 @@ <h1>Eisenhower Task App</h1>
                         <i class="fas fa-plus-square"></i>
                     </button>
                 </div>
-
```

Comment:
## Genereal notes
- move all your files outside of this folder to be directly under src folder because this is going to be a rolling exercise and we don't want it to be on ex1 folder

## To keep
- Github pages is very helpful :)
- cool product additions (e.g the option to mark task as completed)
- cool animations

## To improve
- Note this bug (1)

https://user-images.githubusercontent.com/9549288/169401073-96635c99-3170-42b9-9797-f814a992f98f.mov

- Note this bug (2)

https://user-images.githubusercontent.com/9549288/169401765-b9065c8d-8fd3-4b60-a51c-02d6ae97ae3b.mov

- You have missing requirements - for example ` Change the background and cursor type of an item when hovered on`

I am stopping the review here, please fix the core functionality and then I will review it again @ChipLuxury-EWA